### DISCUSSION TITLE
RPI 5 support

### DISCUSSION THREAD

**Discussion by rutra8002** [home-assistant/operating-system#3139] [Q&A] [answered]
Hey, i was thinking about buying Raspberry PI 5, and i want to use home assistant with it. But as there is currently no stable release for RPI5, should i buy RPI 4 instead, or buy RPI5?

**Comment by groenmarsmannetje** [comment]
I am running already on Pi 5 with pre-release version for weeks now. So to be honest I would always buy the latest version.

**Comment by pedrojreis** [comment, marked as answer]
Hey @rutra8002!

You go ahead and get the Pi 5

[Home assistant OS 12](https://www.home-assistant.io/blog/2024/02/26/home-assistant-os-12-support-for-raspberry-pi-5/) Will provide support to Raspberry 5 😁
